database/migrations/2014_10_12_000000_create_users_table.php

```diff
@@ -19,6 +19,11 @@ public function up()
             $table->string('email')->unique();
             $table->timestamp('email_verified_at')->nullable();
             $table->string('password');
+            $table->integer('role');
+            $table->string('phone')->nullable();
+            $table->date('birthday')->nullable();
+            $table->string('address')->nullable();
+            $table->text('aboutme')->nullable();
```

Comment:
ô không nên chỉnh sửa thẳng cái bảng user vì khi đưa vào sử dụng nếu ô chỉnh sửa bảng user và migrate lại thì toàn bộ dữ liệu của khách hàng sẽ bị mất hết, vì thế nên ô phải tạo thêm một bảng alter_user để thêm dữ liệu vào.
ô có thể tham khảo cách thêm alter table trong đây: https://stackoverflow.com/questions/47516615/how-can-i-add-field-using-alter-table-on-the-migration-laravel